src/local/butler/lint.py

```diff
@@ -15,10 +15,140 @@
 
 import os
 import sys
+import yaml
 
 from local.butler import appengine
 from local.butler import common
 
+_GOLINT_EXCEPTIONS = [
+    'types.go'  # Not all model names conform to Go naming conventions.
+]
+_LICENSE_CHECK_FILENAMES = ['Dockerfile']
+_LICENSE_CHECK_EXTENSIONS = [
+    '.bash',
+    '.c',
+    '.cc',
+    '.cpp',
```

Comment:
I think we are missing `.proto` and `.ps1`

Reply:
done